City-of-Pori/pori-kada
# PORI-278 Added content taxonomy module and configuration to stop new …

…entity creating with autocomplete fields

To test:

1. drush updb -y && drush fra -y && drush cc all
2. Now go edit some page content with keywords. Try adding some keywords that the autocomplete finds and save the page.
3. Now edit the same page again and try adding a keyword that doesn't exist yet. When trying to save the content it should throw an error that you can't create new keywords.
4. Now go edit just about any content type with autocomplete type fields and do the same tests that you did to page content type.
